e2e/terraform/Makefile

```diff
@@ -7,22 +7,22 @@ plan:
 	terraform plan \
 		-var="nomad_local_binary=$(PKG_PATH)" \
 		-var="volumes=false" \
-		-var="client_count_ubuntu_bionic_amd64=2" \
+		-var="client_count_ubuntu_bionic_amd64=3" \
```

Comment:
We have a _lot_ of E2E tests that need at least 3 nodes, including this one, so I've bumped the default here.